composer.json

```diff
@@ -6,10 +6,12 @@
     "require": {
         "behat/behat": "~3.0",
         "behat/mink": "~1.5",
-        "behat/mink-goutte-driver": "~1.0",
         "behat/mink-selenium2-driver": "~1.1",
         "behat/mink-extension": "~2.0"
     },
+    "require-dev": {
+        "behat/mink-goutte-driver": "~1.0"
```

Comment:
this won't really fix our issues, since we have mink-goutte-driver as dev dependency as wll:
```
    "require-dev": {
        "behat/mink-goutte-driver": "^2",
         ...
```